application/src/main/java/org/togetherjava/tjbot/commands/reminder/ReminderCommand.java

```diff
@@ -108,12 +108,18 @@ public void onSlashCommand(SlashCommandInteractionEvent event) {
 
     @Override
     public void onButtonClick(ButtonInteractionEvent event, List<String> args) {
-        int pageNumber = Integer.parseInt(args.get(0));
-        int pageTurnBy = Integer.parseInt(args.get(1));
+        int pageToShow = Integer.parseInt(args.get(0));
+
+        switch (event.getButton().getLabel()) {
+            case PREVIOUS_BUTTON_LABEL -> pageToShow--;
+            case NEXT_BUTTON_LABEL -> pageToShow++;
+            default -> {
+                // do nothing
+            }
```

Comment:
This should never happen so you can just throw an exception here

Reply:
y it should never happen?? u can add more buttons in future like jump to last or jump to first

Reply:
which exception should i throw tho?

Reply:
Im not an expert with exception types but i think an IllegalArgumentException will do the job. I think Zabuzard knows his way around exceptions pretty well though.

Reply:
i used if else now so resolving this